.github/ISSUE_TEMPLATE/ISSUE_REPORT.md

```diff
@@ -48,6 +48,10 @@ Thank you.
 * Python: 
 * Pillow: 
 
+```text
+please paste the output of running `python3 -m PIL --bugreport` here
```

Comment:
Given some of the issues that we've dealt with, I can imagine users running `python3 -m PIL --bugreport` even if they're using a copy of Python other than `python3` to invoke their code.

Would it be better to ask users to paste the output of `from PIL import features;features.pilinfo(supported_formats=False)`?

Reply:
Good point. That also avoids the issue of `--bugreport` not being present in older versions.

However, I think adding the option `python3 -m PIL --bugreport` is still useful on its own.

Reply:
Should that be documented somewhere else then?

Reply:
I've created https://github.com/nulano/Pillow/pull/31

Reply:
Can we suggest both? I like the idea of making it extremely easy as a first suggestion:

---

Paste the output of `python3 -m PIL --bugreport` or the output of the following Python code:
```python
from PIL import features; features.pilinfo(supported_formats=False)
```

Reply:
I'm just concerned about users following `python3 -m PIL --bugreport` so verbatim that they don't change the Python command to what they are using to run their script, e.g. https://github.com/python-pillow/Pillow/discussions/7532#discussioncomment-7507331

However, maybe it is better to make most users lives easier than to cater to absolutely everyone.